Makefile

```diff
@@ -46,14 +46,19 @@ DUMB_ARCH = ${ARCH}
 
 GOBUILD_FLAGS := -v
 
-ALL_ARCH = amd64 arm64
+ALL_ARCH = arm
```

Comment:
I'd have expected you add `arm` in addition, why are you removing existing ones?

Reply:
fixed.